Storage/DoctrineORMStorage.php

```diff
@@ -25,7 +26,7 @@ public function translationsTablesExist()
         $connection = $em->getConnection();
 
         // listDatabases() is not available for SQLite
-        if ('pdo_sqlite' !== $connection->getDriver()->getName()) {
+        if (get_class($connection->getDriver()) !== SQLiteDriver::class) {
```

Comment:
I would use `instanceof here`. Is that not possible in this case?

Reply:
Yes, works the same.

Reply:
Is this a BC break?

Reply:
I think no?